ee/clickhouse/queries/test/test_trends.py

```diff
@@ -134,6 +147,11 @@ def test_breakdown_by_group_props(self):
         )
         response = ClickhouseTrends().run(filter, self.team,)
 
+        with override_instance_config("ENABLE_ACTOR_ON_EVENTS_TEAMS", f"{self.team.pk}"):
+            new_response = ClickhouseTrends().run(filter, self.team)
+
+        self.assertEqual(response, new_response)
```

Comment:
Do we still want this now that we have new testing pattern?

Reply:
Not really. Will get rid of it